hunting/windows/queries/excessive_rdp_network_activity_by_source_host_and_user.toml

```
@@ -0,0 +1,37 @@
+[hunt]
+author = "Elastic"
+description = """
+This hunt looks for a high occurrence of remote desktop connections from the same host and user. The number of unique destination IP addresses is compared to a defined threshold. This could be a sign of discovery or lateral movement via the Remote Desktop Protocol (RDP).
+"""
+integration = ["endpoint", "windows"]
+uuid = "fe01a8a5-6367-4c4c-a57b-be513ab80e42"
+name = "Excessive RDP Network Activity by Host and User"
+language = "ES|QL"
+license = "Elastic License v2"
+notes = [
+    "Further investigation can done pivoting by `host.id` and `user.name`.",
+    "Depending on normal SysAdmin RDP activity, the threshold of 10 can be adjusted to reduce normal noisy activity.",
+    "The second query uses Windows Security log event ID 4624 to summarize numbers of RDP connections by `source.ip` and `user.name` and duration."
+]
+mitre = [ "T1021", "T1021.001",]
```

```suggestion
mitre = [ "T1021", "T1021.001"]
```

Not sure if these need to be all double checked.